shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -1,10 +1,23 @@
-use cassandra_cpp::{stmt, Error, Session, Value, ValueType};
+use cassandra_cpp::{stmt, Cluster, Error, Session, Value, ValueType};
 use ordered_float::OrderedFloat;
 
 mod basic_driver_tests;
 
+#[allow(unused)]
+pub fn cassandra_connection(contact_points: &str, port: u16) -> Session {
```

Comment:
Make sure we use the one from https://github.com/shotover/shotover-proxy/pull/584 before merging